pallets/xyk-liquidity-mining/benchmarking/src/mock.rs

```diff
@@ -344,7 +346,7 @@ impl ExtBuilder {
 		.unwrap();
 
 		pallet_asset_registry::GenesisConfig::<Test> {
-			asset_names: vec![(b"KSM".to_vec(), 1_000), (b"DOT".to_vec(), 1_000)],
+			registered_assets: vec![(b"KSM".to_vec(), 1_000, Some(1)), (b"DOT".to_vec(), 1_000, Some(2))],
```

Comment:
```suggestion
			registered_assets: vec![(b"KSM".to_vec(), 1_000, Some(KSM)), (b"DOT".to_vec(), 1_000, Some(2))],
```
probably introduce `pub const DOT: AssetId = 2;` and use that for the initialization here, too?